### PR TITLE
Feature/add and set default account

### PR DESCRIPTION
**Pull request by cshcomcom**
## Description
- Add 'addAndSetDefaultAccount' function to wallet

### CODE REVIEW

File: src/wallet.ts [hunk 1]
```diff
@@ -78,6 +78,17 @@ export default class Wallet {
     return newAccount.address;
   }
 
+  /**
+   * Adds a new account from the given private key and sets the default account as a new account.
```
Comment (platfowner) [inline src/wallet.ts:82]: s/sets the default account as a new account/sets the new account as the default account/?

File: __tests__/ain.test.ts [hunk 2]
```diff
@@ -102,6 +102,11 @@ describe('ain-js', function() {
       expect(afterLength).toBe(beforeLength);
     });
 
+    it('addAndSetDefaultAccount', function () {
+      ain.wallet.addAndSetDefaultAccount(test_sk);
+      expect(ain.wallet.defaultAccount).not.toBe(null);
```
Comment (liayoo) [inline __tests__/ain.test.ts:107]: Should we check the default account did change to the account corresponding to test_sk?

**Review by liayoo — APPROVED** [review]
LGTM with a comment. Thanks!

File: src/wallet.ts (hunk 1 above)
Comment (cshcomcom) [reply src/wallet.ts:82]: Thanks! I'll change it!

File: __tests__/ain.test.ts (hunk 2 above)
Comment (cshcomcom) [reply __tests__/ain.test.ts]: Looks good! Done

**Comment by cshcomcom** [comment]
Thanks for reviews! Now merging!

**Pull request merged**